packages/components/header/_header-base.scss

```diff
@@ -78,8 +78,9 @@
 }
 
 .nhsuk-header__link {
-  @include nhsuk-logo-size;
   display: block;
+  @include nhsuk-logo-size;
+  @include nhsuk-link-style-white;
```

Comment:
Might need to check how this lands when rebased into the `header-breaking-changes` branch; suspect this might not be needed, but can’t remember.